Description: 'shm_open@@GLIBC_2.3' is defined in DSO //lib/librt.so.1 so try
 adding it to the linker command line
Author: Scott Howard <showard@debian.org>
Bug-Debian: http://bugs.debian.org/618785

Index: sandboxgamemaker-2.8.1/src/Makefile
===================================================================
--- sandboxgamemaker-2.8.1.orig/src/Makefile	2012-11-18 15:08:01.000000000 +0100
+++ sandboxgamemaker-2.8.1/src/Makefile	2012-11-18 15:09:19.446650932 +0100
@@ -24,7 +24,7 @@
 CLIENT_LIBS= -mwindows -Llib -lSDL -lSDL_image -lSDL_mixer -lzdll -lopengl32 -lenet -lws2_32 -lwinmm
 else
 CLIENT_INCLUDES= $(INCLUDES) -I/usr/X11R6/include `sdl-config --cflags`
-CLIENT_LIBS= -lenet -L/usr/X11R6/lib `sdl-config --libs` -lSDL_image -lSDL_mixer -lz -lGL -lX11
+CLIENT_LIBS= -lenet -L/usr/X11R6/lib `sdl-config --libs` -lSDL_image -lSDL_mixer -lz -lGL -lX11 -lrt
 endif
 ifeq ($(PLATFORM),Linux)
 CLIENT_LIBS+= -lrt
